socorro.js

```diff
@@ -51,13 +51,14 @@ if (Object.keys(fileinfo).length != 0) {
         le.element.append(e);
         le.element = e;
       }
-      fetchCoverage(revision, filename).then(data => {
-        if (data !== null && !data.hasOwnProperty("error")) {
+      fetchCoverage(revision, filename).then(data => {console.log(data);
```

Comment:
There's an unintended `console.log` left here

Reply:
oh you see how I'm debugging ;)